OSDOCS#14999: Update the z-stream release notes for 4.19.1 (#95133)
openshift/openshift-docs: release_notes/ocp-4-19-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

// 4.19.1
[id="ocp-4-19-1_{context}"]
=== RHSA-2025:9278 - {product-title} {product-version}.1 image release, bug fix, and security update advisory

Issued: 24 June 2025

{product-title} release {product-version}.1, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:9278[RHSA-2025:9278] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:9279[RHSA-2025:9279] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.19.1 --pullspecs
----

[id="ocp-4-19-1-bug-fixes_{context}"]
==== Bug fixes

> * Previously, a bug was triggered when adding vCenter cloud credentials for the post installation of the {ai-full} because of an invalid ConfigMap object for the cloud provider configuration. As a result, a `missing vcenterplaceholder` error was displayed. With this release, the ConfigMap data is correct, and the error is not displayed. (link:https://issues.redhat.com/browse/OCPBUGS-57384[OCPBUGS-57384])

Reviewer: ```suggestion
* Previously, when you added vCenter cloud credentials for the post installation of the {ai-full}, a bug was triggered because of an invalid `ConfigMap` object for the cloud provider configuration. As a result, a `missing vcenterplaceholder` error was displayed. With this release, the `ConfigMap` data is correct, and the error is not displayed. (link:https://issues.redhat.com/browse/OCPBUGS-57384[OCPBUGS-57384])
```

> * Previously, a network issue during an API call in a cluster caused a timeout in {olmv0-first}. As a result, Operator installations often failed because of timeout issues. With this release, the catalog cache refresh interval is updated to resolve timeout issues. As a result, the likelihood of Operator installation timeouts is reduced. (link:https://issues.redhat.com/browse/OCPBUGS-57352[OCPBUGS-57352])

Reviewer: ```suggestion
* Previously, a network issue during an API call in a cluster caused a timeout in {olmv0-first}. As a consequence, Operator installations often failed because of timeout issues. With this release, the catalog cache refresh interval is updated to resolve timeout issues. As a result, the likelihood of Operator installation timeouts is reduced. (link:https://issues.redhat.com/browse/OCPBUGS-57352[OCPBUGS-57352])
```
I swapped out "result" for "consequence" to avoid having two instances of "As a result" near each other in the same paragraph.

> * Previously, Operator group reconciliation in {olmv0-first} triggered unnecessary `ClusterRole` updates because of the changing order of aggregation rule selectors. As a result, unnecessary API server writes occurred. With this release, a bug fix ensures the deterministic order of `ClusterRoleSelectors` in the aggregation rule, reducing unnecessary API server writes and improving cluster stability. (link:https://issues.redhat.com/browse/OCPBUGS-57279[OCPBUGS-57279])

Reviewer: I'm not sure what to suggest, but it would be good to add a noun after `ClusterRoleSelectors" to comply with this guidance from the [OCP docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references): "Use "object", "resource", "custom resource", "spec", etc. as appropriate after the object reference. This helps with clarity and readability."

Author: Added 'array' per https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/rbac_apis/clusterrole-rbac-authorization-k8s-io-v1#aggregationrule

> * Previously, ignoring the `AdditionalTrustBundlePolicy` in assisted-service's installation configuration caused users to incorrectly set the `AdditionalTrustBundlePolicy`, affecting Federal Information Processing Standard (FIPS) and other installation configuration overrides. With this release, the installation configuration includes an `AdditionalTrustBundlePolicy` field, ensuring overrides are applied correctly. Users can correctly set the `AdditionalTrustBundlePolicy` field, and ensure that FIPS and other installation configuration overrides function as intended. (link:https://issues.redhat.com/browse/OCPBUGS-57208[OCPBUGS-57208])

Reviewer: ```suggestion
* Previously, the `AdditionalTrustBundlePolicy` setting in the assisted-service installation configuration led to Federal Information Processing Standard (FIPS) and other installation-configuration overrides. With this release, the installation configuration includes an `AdditionalTrustBundlePolicy` field, which you can set to ensure that FIPS and other installation-configuration overrides function as intended. (link:https://issues.redhat.com/browse/OCPBUGS-57208[OCPBUGS-57208])
```

* Previously, the authentication process for the `/metrics` endpoint was missing a token review check and caused unauthorized requests. As a result, the {product-title} console was prone to `TargetDown` alerts. With this release, the token review for unauthorized requests occurs with the user token provided in the request context. As a result, unauthorized requests to the {product-title} console do not cause `TargetDown` alerts. (link:https://issues.redhat.com/browse/OCPBUGS-57180[OCPBUGS-57180])

> * Previously, the `Started` column was hidden because of reduced screen size. This caused the `VirtualizedTable` component to malfunction because of a missing sort function. As a result, the table sorting functionality was affected in the `PipelineRun` list pages. With this release, the table component handles missing sort functions correctly for reduced screen sizes. (link:https://issues.redhat.com/browse/OCPBUGS-57110[OCPBUGS-57110]) 

Reviewer: ```suggestion
* Previously, the *Started* column was hidden when screen size was reduced. As a consequence, the `VirtualizedTable` component malfunctioned because of a missing sort function, and the table sorting functionality was affected on the *PipelineRun* list pages. With this release, the table component handles missing sort functions correctly for reduced screen sizes. (link:https://issues.redhat.com/browse/OCPBUGS-57110[OCPBUGS-57110]) 
```

> * Previously, users did not configure a default masthead logo when customizing the theme. This omission caused an inconsistency in the user interface. With this release, the masthead logo displays a default option for both light and dark themes, improving the interface consistency. (link:https://issues.redhat.com/browse/OCPBUGS-57054[OCPBUGS-57054])

Reviewer: ```suggestion
* Previously, if you configured a masthead logo for a theme but used the default settings for the rest of the theme, the logo shown on the user interface was inconsistent. With this release, the masthead logo displays a default option for both light and dark themes, improving the interface consistency. (link:https://issues.redhat.com/browse/OCPBUGS-57054[OCPBUGS-57054])
```

> * Previously, the cluster installation failed because of an invalid security group configuration for  a Network Load Balancer (NLB). This failure prevented the traffic from both primary subnets for bootstrapping. With this release, the security group is updated to allow traffic from both primary subnets for bootstrapping, and the cluster installation does not fail because of security group restrictions on additional primary subnets. (link:https://issues.redhat.com/browse/OCPBUGS-57039[OCPBUGS-57039])

Reviewer: ```suggestion
* Previously, the cluster installation failed because of an invalid security group configuration for a Network Load Balancer (NLB). This failure prevented the traffic from both primary subnets for bootstrapping. With this release, the security group allows traffic from both primary subnets for bootstrapping, and the cluster installation does not fail because of security group restrictions on additional primary subnets. (link:https://issues.redhat.com/browse/OCPBUGS-57039[OCPBUGS-57039])
```

> * Previously, users without project access saw an incomplete roles list because of improper API group access. As a result, users saw partial role information on the *Roles* page. With this release, users without project access cannot see an incomplete roles list on the *Roles* page. (link:https://issues.redhat.com/browse/OCPBUGS-56987[OCPBUGS-56987])

Reviewer: ```suggestion
* Previously, users without project access saw an incomplete roles list on the *Roles* page because of improper API group access. With this release, users without project access cannot see an incomplete roles list on the *Roles* page. (link:https://issues.redhat.com/browse/OCPBUGS-56987[OCPBUGS-56987])
```
Do we need to say what users without project access can see instead of the partial list?

> * Previously, the `node-image create` command modified directory permissions and caused user directories to lose original permissions during the operation. With this release, the `node-image create` command preserves file permissions during `rsync`, and ensures that user directories maintain original permissions during the operation. (link:https://issues.redhat.com/browse/OCPBUGS-56905[OCPBUGS-56905])

Reviewer: I'm not sure what to suggest, but it would be good for clarity to add a noun after `rsync`.

> * Previously, the `delete` keyword in image names was allowed for `ImageSetConfiguration`, which is not supported. As a consequence, users encountered errors while mirroring images. With this release, the error for image names ending with `delete` in `ImageSetConfiguration` has been removed. As a result, users can now successfully mirror images with names ending in `delete`. (link:https://issues.redhat.com/browse/OCPBUGS-56798[OCPBUGS-56798])

Reviewer: For clarity purposes, add a noun after each instance of `ImageSetConfiguration`.

> * Previously, the user interface in the `Observe Alerting` field displayed incorrect alert severity icons for information alerts. With this release, the alert severity icons match in the `Observe Alerting` field. As a result, alert icons match consistently, reducing potential confusion for users. (link:https://issues.redhat.com/browse/OCPBUGS-56470[OCPBUGS-56470])

Reviewer: ```suggestion
* Previously, the *Observe Alerting* field displayed incorrect alert severity icons for information alerts. With this release, the alert severity icons match in the *Observe Alerting* field. As a result, alert icons match consistently, reducing potential confusion. (link:https://issues.redhat.com/browse/OCPBUGS-56470[OCPBUGS-56470])
```

> * Previously, using an unauthorized access configuration file in the `oc-mirror` command triggered an error. As a result, users encountered an `Unauthorized `error while synchronizing their image sets. With this release, the Docker configuration is updated to use a custom authorization file for authentication. Users can successfully synchronize their image sets without encountering the `Unauthorized` error. (link:https://issues.redhat.com/browse/OCPBUGS-55701[OCPBUGS-55701])

Reviewer: ```suggestion
* Previously, if you used an unauthorized access configuration file in the `oc-mirror` command, an `Unauthorized` error was displayed when you synchronized your image sets. With this release, the Docker configuration is updated to use a custom authorization file for authentication. You can successfully synchronize your image sets without encountering the `Unauthorized` error. (link:https://issues.redhat.com/browse/OCPBUGS-55701[OCPBUGS-55701])
```
